tests/MEM/mem.sh

```diff
@@ -0,0 +1,8 @@
+export LD_LIBRARY_PATH=../..
```

Comment:
This and vm.sh

1. aren't called anywhere, is that happening in another pr that will hook up .github/workflows to do CI?
2. aren't doing set -e or a #!/bin/sh